### PR TITLE
Enabled jittering by default

### PR DESCRIPTION
**Pull request by asmoscosa**
#851

### CODE REVIEW

File: tomviz/ModuleVolume.cxx [hunk 1]
```diff
@@ -191,6 +192,7 @@ void ModuleVolume::addToPanel(QWidget* panel)
   QCheckBox* maxIntensity = new QCheckBox("Maximum intensity");
   layout->addWidget(maxIntensity);
   QCheckBox* jittering = new QCheckBox("Jittering");
+  jittering->setChecked(true);
```
Comment (cryos) [inline tomviz/ModuleVolume.cxx]: You shouldn't need this line as line 202 sets it checked based upon the state of m_volumeMapper

Comment (asmoscosa) [inline tomviz/ModuleVolume.cxx]: Removed the line.

**Comment by cjh1** [comment]
@asmoscosa To remove these python 3 errors, rebase on master :smile:

**Comment by cryos** [comment]
Thanks!

**Pull request merged**